### ISSUE TITLE
seems like this plugin doesn't respect FZF_DEFAULT_COMMAND

### ISSUE THREAD

**Issue by isvicy** [nvim-telescope/telescope-fzf-writer.nvim#4]
by default, telescope-fzf-writer doesn't show hidden files.

set FZF_DEFAULT_COMMAND works for fzf.vim but not for telescope-fzf-writer.

Is this intensional? and how can i make fzf-writer show hidden files?

**Comment by elianiva** [comment]
yeah, it seems like it's hardcoded. I don't think it would be that hard to make it respect `$FZF_DEFAULT_COMMAND`

https://github.com/nvim-telescope/telescope-fzf-writer.nvim/blob/9535863f519be3d5e57fd50916f96594241bfe16/lua/telescope/_extensions/fzf_writer.lua#L131-L132

**Comment by Conni2461** [comment]
Plenary job has a clean env. So to make this work we need to do `os.getenv` and pass in that `env` to plenary job. Should be not hard. We might need some changes in core i am not sure how finder handles env
